File: c/Natural.c
#include <stdio.h>

int main(void) {
  int Count, Counter = 1;
  printf("How many natural numbers you want to print?");
  scanf("%d", &Count);
  printf("The natural numbers upto %d are", Count);
  for(Counter = 1; Counter <= Count; Counter++)
  printf(" %d", Counter);
  printf(".");
  return 0;
}

File: swift/Natural.swift
print("How many natural numbers you want to print?", terminator : " ");
var Count = Int(readLine()!)!
var Counter = 1;
print("The natural numbers upto \(Count) are", terminator : " ");
while(Counter <= Count) {
 print(" \(Counter)", terminator : "");
 Counter = Counter + 1;
}
print(".");